modules/GanneffServ.rb

```diff
@@ -648,6 +744,21 @@ def load_data()
       count += 1
     }
     result.free
+
+    @protection = Hash.new
+    result = DB.execute(@dbq['GET_PROTECTED_PATTERNS'], '')
+    count = 0
+    result.row_each { |row|
+      pattern = row[0]
+      reason = row[1]
+      pattern = irc_pattern_to_regex(pattern)
+      pattern = Regexp.escape(pattern)
```

Comment:
Why `Regexp.escape()` here and nowhere else? I'm guessing this is vestigial (but breaks things).

Reply:
😳